packages/site-kit/src/lib/markdown/renderer.ts

```diff
@@ -283,6 +283,24 @@ export async function render_content_markdown(
 				// Save everything locally now
 				snippets.save(token.text, html);
 			}
+
+			// ensure that `foo`/`bar` is transformed to `foo` / `bar`
+			// https://github.com/sveltejs/svelte.dev/pull/577
+			const slash_index =
+				// @ts-expect-error
+				token.tokens?.findIndex((token) => token.type === 'text' && token.text === '/') ?? -1;
+
+			if (slash_index !== -1) {
+				// @ts-expect-error
+				const before = token.tokens[slash_index - 1];
+				// @ts-expect-error
+				const after = token.tokens[slash_index + 1];
+
+				if (before?.type === 'codespan' && after?.type === 'codespan') {
+					// @ts-expect-error
+					token.tokens[slash_index].raw = token.tokens[slash_index].text = ' / ';
+				}
+			}
```

Comment:
Do we actually want to check in all these changes if we're doing this transformation?

Reply:
good point